hdfs/util.py

```diff
@@ -115,6 +115,10 @@ def seekable(self):
     """
     return False
 
+  def __iter__(self):
```

Comment:
Rather than add an invalid `__iter__` method, can we set `__iter__ = None`? It's slightly cleaner since the class doesn't support iteration, and looks like it would still address the underlying issue.

Reply:
Yes you are right, i'm making the change